docs/guide/arktos_mizar_installation_single.md

```diff
@@ -0,0 +1,140 @@
+<!--
+SPDX-License-Identifier: MIT
+Copyright (c) 2020 The Authors.
+
+Authors: Catherine Lu      <@clu2>

```

Comment:
better change to team, if it is not against the setup convention.

Reply:
I can add individual names. This is the convention mizar follows for all files